resources/python/pipelines/pipelines.yml

```diff
@@ -0,0 +1,179 @@
+# Copyright 2023 Autodesk, Inc.  All rights reserved.
+#
+# Use of this software is subject to the terms of the Autodesk license agreement
+# provided at the time of installation or download, or which otherwise accompanies
+# this software in either electronic or hard copy form.
+
+parameters:
+  branch: ""
+
+jobs:
+- job: install_source_dependencies_3_7
+  displayName: Install source dependencies Python 3.7
+  condition: not(endsWith( variables['System.PullRequest.SourceBranch'], '-automated'))
+  pool:
+    vmImage: 'macOS-12'
+  steps:
+  - template: template.yml
+    parameters:
+      python_version: 3.7
+  - script: |
+      git checkout $(System.PullRequest.SourceBranch)
+      git checkout -b ${{ parameters.branch }}
+      python update_requirements.py
+      ./install_source_only.sh
+      git commit -am "Update source requirements 3.7"
+      git push -u origin ${{ parameters.branch }}
+    displayName: Run Scripts
+    workingDirectory: resources/python
+
+- job: install_source_dependencies_3_9
+  displayName: Install source dependencies Python 3.9
+  dependsOn: install_source_dependencies_3_7
+  condition: not(endsWith( variables['System.PullRequest.SourceBranch'], '-automated'))
+  pool:
+    vmImage: 'macOS-12'
+  steps:
+  - template: template.yml
+    parameters:
+      python_version: 3.9
+      branch: ${{ parameters.branch }}
+  - script: |
+      git checkout ${{ parameters.branch }}
+      python update_requirements.py
+      ./install_source_only.sh
+      git pull
```

Comment:
Why do you pull here? If we are not synced, should fetch earlier instead?

Reply:
I think I'm going to remove that line. When I was implementing this I had some race conditions, but now the jobs are dependent from each other.